code/modules/core_implant/cruciform/rituals/path.dm

```diff
@@ -250,6 +250,44 @@
 	phrase = "Dominus, ait, in cujus conspectu ambulo, mittet angelum suum tecum, et diriget viam tuam." //"He replied, ‘The Lord, before whom I have walked faithfully, will send his angel with you and make your journey a success.'"
 	stats_to_boost = list(STAT_ROB = 15, STAT_TGH = 15, STAT_VIG = 15)
 
+/datum/ritual/cruciform/lemniscate/holy_boost
+	name = "Bolster the Fellowship"
+	phrase = "Benedicat tibi Dominus, et custodiat te. Ostendat Dominus faciem suam tibi, et misereatur tui. Convertat Dominus vultum suum ad te, et det tibi pacem." //“The Lord bless you and keep you; the Lord make his face shine on you and be gracious to you; the Lord turn his face toward you and give you peace.”
+	var/stats_to_boost = list(STAT_MEC = 15, STAT_COG = 15, STAT_BIO = 15, STAT_ROB = 15, STAT_TGH = 15, STAT_VIG = 15)
+	desc = "This litany boosts all the stats of all disciples who hear you for thirty minutes, but has no effect on those without cruciforms."
+
+/datum/ritual/cruciform/lemniscate/holy_boost/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/cruciform/C)
+	var/list/people_around = list()
+	for(var/mob/living/carbon/human/H in view(user))
+		if(H != user && !isdeaf(H) && is_neotheology_disciple(H)) //This one only for the homies
+			people_around.Add(H)
+
+	if(people_around.len > 0)
+		to_chat(user, SPAN_NOTICE("Your feel the air thrum with an inaudible vibration."))
+		playsound(user.loc, 'sound/machines/signal.ogg', 50, 1)
+		for(var/mob/living/carbon/human/participant in people_around)
+			to_chat(participant, SPAN_NOTICE("You hear a silent signal..."))
+			give_boost(participant)
+			add_effect(participant, FILTER_HOLY_GLOW, 25)
+		set_global_cooldown()
+		return TRUE
+	else
+		fail("Your cruciform sings, alone, unto the void.", user, C)
+		return FALSE
```

Comment:
```suggestion
	fail("Your cruciform sings, alone, unto the void.", user, C)
	return FALSE
```

Reply:
While this would return the same result, the way I did it is how it's done in most of the other litanies. Just easier to stay consistent and readable

Reply:
Sometime i should go through and standerise all litanies internal code to be a bit more streamlined